src/com/yetanalytics/lrs_admin_ui/subs.cljs

```diff
@@ -326,3 +325,9 @@
  :<- [:status/loading-map]
  (fn [loading-map [_ loading-k]]
    (true? (get loading-map loading-k))))
+
+;; Delete Actor
+(reg-sub
+ :delete-actor/enabled?
+ (fn [db _]
+   (::db/enable-admin-delete-actor db false)))
```

Comment:
i have never seen this notation ever, but could be my ignorance. Is that false a default for nils? I can't even find documentation on this.

Reply:
![image](https://github.com/yetanalytics/lrs-admin-ui/assets/334165/21230418-2f2e-42dc-86fd-7bc5e036fb37)

keywords-as-fns are just calls to `get` under the hood, where it's documented

Reply:
iiinteresting!